UKHomeOffice/sgmr-service
# Update time fields

### AC / Description of changes
This PR updates the time fields to be in line with recent changes made to the BE. `departureTime` and `arrivalTime` should now reference `departureTimeFrom` and `arrivalTimeFrom` instead.

### To Test
* Pull branch and run tests, linter
* All should pass
* Create a new voyage plan and fill out the departure and arrival forms, clicking save and continue after each one
* You should be able to enter the departure and arrival times without a problem
* Edit an existing voyage plan
* You should be able to edit the departure and arrival times without a problem
* Review code changes, ensure best practices are followed, and that there are no mistakes or redundant/unnecessary code modifications

### Notes

---

**Remember to merge to feature branches not `master`.**
